src/overlays/actors/ovl_Dm_Char07/z_dm_char07.h

```diff
@@ -5,9 +5,31 @@
 
 struct DmChar07;
 
+typedef void (*DmChar07ActionFunc)(struct DmChar07*, GlobalContext*);
+
+#define DMCHAR07_GET_SPOTLIGHTFLAGS(thisx) (((thisx)->params >> 8) & 0x1F)
+#define DMCHAR07_GET_TYPE(thisx) ((thisx)->params & 0xFF)
+
+#define DMCHAR07_STAGE_RIGHT_SPOTLIGHT(this) ((this)->spotlightFlags & 1)   // Deku spotlight
+#define DMCHAR07_UPSTAGE_RIGHT_SPOTLIGHT(this) ((this)->spotlightFlags & 2) // Goron spotlight
+#define DMCHAR07_STAGE_CENTER_SPOTLIGHT(this) ((this)->spotlightFlags & 4)  // Human spotlight
+#define DMCHAR07_UPSTAGE_LEFT_SPOTLIGHT(this) ((this)->spotlightFlags & 8)  // Zora spotlight
+#define DMCHAR07_DOWNSTAGE_CENTER_SPOTLIGHT(this) ((this)->spotlightFlags & 16)
+
+typedef enum {
+    /* 0x00 */ STAGE,
+    /* 0x01 */ CREDITS_STAGE,
+    /* 0x02 */ SPOTLIGHTS,
+    /* 0x03 */ MISC_OBJECTS
```

Comment:
I'm not quite sure what to call this. Anyone got any better ideas?
![image](https://user-images.githubusercontent.com/31974442/121286618-efdec080-c894-11eb-9d57-00cdd2258012.png)

Reply:
`DMCHAR07_MILK_BAR_DISHES`? Looks like a plate with three bottles of milk